### PR TITLE
[Compute] vm/vmss/availability-set update: add --ppg to allowing updating ProximityPlacementGroup

### PR DESCRIPTION
**Pull request by qwordy**
Issue #11420 

**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by jiasli** [comment]
`--ppg` is missing help.

```
az vm availability-set update -h

Command
    az vm availability-set update : Update an Azure Availability Set.

Arguments
    --ppg
```

**Comment by yonzhan** [comment]
add to S165.

File: src/azure-cli/azure/cli/command_modules/vm/custom.py [hunk 1]
```diff
@@ -1098,6 +1098,9 @@ def update_vm(cmd, resource_group_name, vm_name, os_disk=None, disk_caching=None
         else:
             vm.billing_profile.max_price = max_price
 
+    if proximity_placement_group is not None:
+        vm.proximity_placement_group = {'id': proximity_placement_group}
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/vm/custom.py]: Is there a model for proximity_placement_group?

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/vm/custom.py]: in params.py:
with self.argument_context(scope, min_api='2018-04-01')

**Pull request merged**